vision/CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 [1]: https://pypi.org/project/google-cloud-vision/#history
 
+## 0.40.0
+
+11-08-2019 09:30 PST
+
+
+### Implementation Changes
+- Add proto annotations (via synth) ([#9441](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/9441))
```

Comment:
```suggestion
- Add proto annotations (via synth). ([#9441](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/9441))
```

Reply:
I know we have to leave the periods off to appease conventionalcommits, but that presumes that our tooling (release-please?) adds it back.